### ISSUE TITLE
PostsPerPageSniff.php does not catch the `-1` if the line is not followed by a comma 

### ISSUE THREAD

**Issue by GaryJones** [WordPress/WordPress-Coding-Standards#1505]
Original Issue: https://github.com/Automattic/VIP-Coding-Standards/issues/29

Rejected PR: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1297

The sniff catches following:

```
<?php

$args = array(
    'posts_per_page' => -1,
);
```

but not following:

```
<?php

$args = array(
    'posts_per_page' => -1
);
```

Reporting here as a reminder to investigate and reproduce this.

**Comment by dingo-d** [comment]
This is no longer reproducible since PR #1260 and the VIP part was removed in #1551. So we will close this issue.

**Comment by jrfnl** [comment]
This is actually a real bug, which was also reported in #2211 and will be fixed by open PR #2266.
